src/class_singleWorker.py

```diff
@@ -226,6 +231,18 @@ def _doPOWDefaults(self, payload, TTL,
         # inventoryHash = calculateInventoryHash(payload)
         return payload
 
+    def sendRawObject(self, data):
```

Comment:
How is this method related to sending any object? Where that objects get out of the queue?

Reply:
It's inserted into the Inventory(). What is missing for full functionality, the hash also needs to be inserted into the invQueue.

Reply:
occurrences of `broadcastToSendDataQueues` in `protocol.py`  should be replaced with `invQueue.put((stream, inventoryHash))` or something like that.